main/webapp/modules/core/scripts/index/parser-interfaces/preview-table.js

```diff
@@ -104,7 +106,22 @@ Refine.PreviewTable.prototype._render = function() {
           .attr("target", "_blank")
           .appendTo(divContent);
         } else {
-          $('<span>').text(cell.v).appendTo(divContent);
+          if ($.cookie('preference_control_char') == 'true' ) {
+            var stringIncNonPrintable = "";
+            for (var character = 0; character < cell.v.length; character++) {
+              var unprintableChar = "";
+              var charCode = cell.v.charAt(character).charCodeAt(0);
+              if (charCode <= 32) {
+                var size = (controlCharacters[charCode].length + 2) * 2;
+                unprintableChar = "<input type='button' class='unprintableCharacters' disabled='disabled' width=" + size + "% value=" + controlCharacters[charCode] + ">";
```

Comment:
To me it looks much better if change `unprintableCharacters` to have a `style="padding-left: 2px;padding-right: 2px;"` added.  Otherwise it seems there's a default padding of `8` on both sides and makes it too stretched?

Reply:
Sure. I will update the styling.